src/shmem_internal.h

```diff
@@ -25,22 +25,21 @@
 #include "shmemx.h"
 #include "runtime.h"
 #include "config.h"
+#include "shmem_env.h"
 
 extern int shmem_internal_my_pe;
 extern int shmem_internal_num_pes;
 
 extern int shmem_internal_initialized;
 extern int shmem_internal_finalized;
 extern int shmem_internal_thread_level;
-extern int shmem_internal_debug;
-extern int shmem_internal_trap_on_abort;
 
 extern void *shmem_internal_heap_base;
 extern long shmem_internal_heap_length;
 extern void *shmem_internal_data_base;
 extern long shmem_internal_data_length;
-extern int shmem_internal_heap_use_huge_pages;
-extern long shmem_internal_heap_huge_page_size;
+
+#define SHMEM_INTERNAL_LIBRARY_OVERHEAD (1024*1024)
```

Comment:
Maybe this could be "MEMORY" or "HEAP" OVERHEAD?  It's not a big deal, but it took a moment to see what this is.  At first I thought it could be a time quantum.